### DISCUSSION TITLE
Setting Connection Options in Netbox

### DISCUSSION THREAD

**Discussion by MrPaulAR** [scrapli/nornir_scrapli#141] [Q&A] [answered]
Is there a mechanism to set the ConnectionOptions for scrapli in netbox?

Right now I'm doing this in code but I think it would be better set within netbox.   Specifically the classic variant.  The other fields are being pulled from the inventory/defaults.yml file but I'm lazilly setting it again.

``` python
from nornir.core.inventory import ConnectionOptions
nokia_rtrs.inventory.defaults.connection_options["scrapli"] = ConnectionOptions(extras={'auth_strict_key': False,
    'ssh_config_file': True, 'variant': "classic"})
```

**Comment by carlmontanari** [comment, marked as answer]
no idea! I imagine this is more a net box inventory question than a scrapli thing? if the inventory plugin is setting things for other transports I dont see why it couldnt/wouldnt work since the options for scrapli connections are all just set normally via the extras and such like "normal" nornir things!

**Reply by MrPaulAR** [reply]
Good point.  I'll move my question there.

**Reply by carlmontanari** [reply]
sounds good! if you get an answer it'd be awesome if ya can drop it here as well! thanks @MrPaulAR !

**Comment by MrPaulAR** [comment]
For anyone following along check https://github.com/wvandeun/nornir_netbox/issues/44
